server.js

```diff
@@ -57,10 +61,10 @@ app.delete('/books/:id', async (request, response) => {
   try {
     await Book.findByIdAndDelete(id);
     response.status(202).send('Book Succesfully Burned')
-  } catch(error) {
-    console.log('---> DELETE BOOKS ERROR LOG <---');
+  } catch (error) {
+    // console.log('---> DELETE BOOKS ERROR LOG <---');
     response.status(500).send('No Books to Burn!')
```

Comment:
Nice work on this, good progress made!

Feedback here: Consider doing something with the error in catch, you can perhaps use:
` return res.status(error code).json({ error: error.toString() });`

-  or `res.json({ message: error.message })` or:

` res.json({
    status: error.status,
    message: error.message,
`